### ISSUE TITLE
Make tests for do_path_action

### ISSUE THREAD

**Issue by ellychoi** [uchicago-cs/chiventure#181]
The object of this issue is to make multiple tests for do_path_action, covering all possible outcomes of the function in order to assure that it works correctly. Since the functions at this time are still fairly simple, the outputs of this tests will show whether they are printing the correct strings or not.

This was a branch off issue #177

**Comment by SnowballAntrobus** [comment]
Created task #191 to add new tests after function has been updated to use game state functions.

**Comment by SnowballAntrobus** [comment]
There is now a PR for this issue into dev #243 .

**Comment by SnowballAntrobus** [comment]
Closing statement:
We wrote basic tests for the function do_path_action. These tests focused on making sure that the correct string is being printed when correct and incorrect action kinds are used. In future work we would make testing simpler by making the return value of the function an error code int. Additionally, we will make more tests for this function once it is updated to change game state.
